appcelerator-archive/amp
# Stack remove using name or fragment IDs

Resolves #1103 

## How to test
```
$ ampmake build-server build-cli
$ alias amp='amp -s localhost'
$ amp cluster create --tag=local
$ amp stack ls
[Should be empty]
$ amp stack deploy -c examples/stacks/pinger/pinger.yml pinger
[localhost:50101]
Deploying stack pinger using examples/stacks/pinger/pinger.yml
Updating service pinger_pinger (id: s3bgn8gi22mz85e2u00fx6z95)
unable to pin image subfuzion/pinger to digest: manifest unknown: manifest unknown

$ amp stack ls
[Get stack id]
$ amp stack rm fragment
[localhost:50101]
Removing service pinger_pinger

$ amp stack deploy -c examples/stacks/pinger/pinger.yml pinger
[localhost:50101]
Deploying stack pinger using examples/stacks/pinger/pinger.yml
Updating service pinger_pinger (id: s3bgn8gi22mz85e2u00fx6z95)
unable to pin image subfuzion/pinger to digest: manifest unknown: manifest unknown

$ amp stack rm pinger
[localhost:50101]
Removing service pinger_pinger

```
